data/os/RedHat/8.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Might make more sense to make defaults in `RedHat.yaml` be for RHEL8 and have the overrides be for RHEL7 in `7.yaml`.  That way when RHEL9 comes out it's likely the 8 and 9 changes will be similar